redis-lettuce/src/main/java/io/micronaut/configuration/lettuce/cache/RedisCache.java

```diff
@@ -230,12 +278,47 @@ public void put(Object key, Object value) {
         putValue(serializedKey, value);
     }
 
+    public void put(Map<?, ?> values) {
+        if (CollectionUtils.isNotEmpty(values)) {
+            Map<byte[], byte[]> toSave = new HashMap<>(values.size());
```

Comment:
what is the map is a `LinkedHashMap` with order? Does it matter if order can be lost here?

Reply:
`toSave` is only used locally, to call `mset`, where order doesn't matter.